### PR TITLE
INTYGFV-15642: Added new DB table REKO with related columns.

### PR DESCRIPTION
**Pull request by ChrilleSigerhed**

### CODE REVIEW

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
Introduce RekoRepository as well!

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
A few comments to consider.

File: persistence/src/test/java/se/inera/intyg/intygstjanst/persistence/model/dao/impl/RekoRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import java.util.stream.Collectors;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import se.inera.intyg.intygstjanst.persistence.config.JpaConstants;
+import se.inera.intyg.intygstjanst.persistence.model.dao.Reko;
+import se.inera.intyg.intygstjanst.persistence.model.dao.RekoRepository;
+
+public class RekoRepositoryTest extends TestSupport {
```
[the remark this answers is not in the document]
Comment (ChrilleSigerhed) [reply persistence/src/test/java/se/inera/intyg/intygstjanst/persistence/model/dao/impl/RekoRepositoryTest.java:34]: Good idea, done!

File: persistence/src/main/java/se/inera/intyg/intygstjanst/persistence/model/dao/RekoRepository.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface RekoRepository extends JpaRepository<Reko, Long> {
+
+    List<Reko> findByPatientId(String patientId);
+
+    List<Reko> findByPatientIdIn(List<String> patientIds);
+
+    List<Reko> findByPatientIdInAndCareUnitId(List<String> patientId, String careUnitId);
```
Comment (mww-sopra-steria) [inline persistence/src/main/java/se/inera/intyg/intygstjanst/persistence/model/dao/RekoRepository.java:33]: patientId -> patientIds

**Review by mww-sopra-steria — APPROVED** [review]
Nice! Approved! You could rename an argument if you like, before merging... see comment.

**Pull request merged**